### PR TITLE
Adding WN container docs

### PR DESCRIPTION
**Pull request by djw8605**
Should I add all of the configuration changes required for running jobs in a docker container?  There's a lot of changes.

### CODE REVIEW

File: docs/worker-node/using-wn-containers.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Available tags include:
+
+* `latest`: The latest version of the OSG worker node environment on the most recent supported OS.  As of June 2017, this is OSG 3.4 and RHEL7.
+* `3.4`: The OSG 3.4 release series on top of the most recent supported OS.  As of June 2017, this is RHEL7.
+* `3.4-el7`: The OSG 3.4 release series on top of a RHEL7 environment.
+* `3.4-el6`: The OSG 3.4 release series on top of a RHEL6 environment.
+* `3.3`: The OSG 3.3 release series on top of the most recent supported OS.  As of June 2017, this is RHEL7.
+* `3.3-el7`: The OSG 3.3 release series on top of a RHEL7 environment.
+* `3.3-el6`: The OSG 3.3 release series on top of a RHEL6 environment.
```
Comment (brianhlin) [inline docs/worker-node/using-wn-containers.md]: 3.3 isn't supported so let's not list all of those tags

Comment (djw8605) [reply docs/worker-node/using-wn-containers.md]: Sure, I'll remove them from the README on the docker repo as well.

**Comment by djw8605** [comment]
Yes, setting up docker universe routing.  And then the changes required on the worker nodes.

**Comment by brianhlin** [comment]
Just curious, what's the routing look like for non-HTCondor batch systems?

Changes required on the worker nodes being installation of docker? Anything else?

**Comment by djw8605** [comment]
Actually, I'm not sure what the routing would be for non-htcondor.  It would probably include some stuff in the `*_local_submit_attributes.sh` file.  Maybe some large changes to the actual `blahp` in order to execute the job inside docker.  And changes in, for example, SLURM, which does not have native support for Docker.

Right, installation of docker.  And the configuration of docker in HTCondor.  For example, here is Nebraska's HTCondor docker config below.  The Nebraska CMS T2 runs all jobs in their own Docker container.  Lots of config to include CVMFS into the container, ca certs, authorization stuff, distributed filesystem...

```
START = ((TARGET.WantDocker =?= true) || (TARGET.IsT3User =?= true)) && $(START)

DOCKER_VOLUMES = CVMFS, ETC_CVMFS, HDFS, GRID_SECURITY, LCMAPS, SSSD, NSSWITCH, DEV_LOG

DOCKER_VOLUME_DIR_CVMFS         = /cvmfs:/cvmfs:ro,slave
DOCKER_VOLUME_DIR_ETC_CVMFS     = /etc/cvmfs:/etc/cvmfs:ro
DOCKER_VOLUME_DIR_HDFS          = /mnt/hadoop:/mnt/hadoop:ro
DOCKER_VOLUME_DIR_GRID_SECURITY = /etc/grid-security:/etc/grid-security:ro
DOCKER_VOLUME_DIR_LCMAPS        = /etc/lcmaps.db.docker:/etc/lcmaps.db:ro
DOCKER_VOLUME_DIR_SSSD_PIPES    = /var/lib/sss/pipes
DOCKER_VOLUME_DIR_SSSD_MC       = /var/lib/sss/mc:/var/lib/sss/mc:ro
DOCKER_VOLUME_DIR_NSSWITCH      = /etc/nsswitch.conf:/etc/nsswitch.conf:ro
DOCKER_VOLUME_DIR_DEV_LOG       = /dev/log

# TODO: Distribute lcmaps.db with puppet
DOCKER_MOUNT_VOLUMES = CVMFS, ETC_CVMFS, HDFS, GRID_SECURITY, SSSD_PIPES, SSSD_MC, NSSWITCH, DEV_LOG

# Dropping kernel capabilities (docker run --cap-drop=all) breaks glExec
DOCKER_DROP_ALL_CAPABILITIES = false

# Use wrapper script to add capabilities when running trusted images
# Adding CAP_SYS_ADMIN enables mount() for glExec and Singularity bind mounts
DOCKER = /usr/local/libexec/condor-docker
```

**Comment by djw8605** [comment]
Ok, removed mentions of running jobs within docker containers.

**Comment by brianhlin** [comment]
JIRA ticket created for documenting how to run jobs in the WN client container: https://opensciencegrid.atlassian.net/browse/SOFTWARE-3590

**Pull request merged**